lib/osf-components/addon/components/node-card/component.ts

```diff
@@ -63,18 +79,45 @@ export default class NodeCard extends Component {
             registration.reviewsState === RegistrationReviewStates.Embargo
         ) {
             try {
-                const revisions = await registration.queryHasMany('schemaResponses');
+                const revisions = await registration.queryHasMany(
+                    'schemaResponses',
+                );
                 if (revisions) {
                     this.latestSchemaResponse = revisions[0];
                 }
             } catch (e) {
-                const errorMessage = this.intl.t('node_card.schema_response_error');
-                captureException(e, {errorMessage});
+                const errorMessage = this.intl.t(
+                    'node_card.schema_response_error',
+                );
+                captureException(e, { errorMessage });
                 this.toast.error(getApiErrorMessage(e), errorMessage);
             }
         }
     }
 
+    @task
+    @waitFor
+    async getGuidMetadata() {
+        const guidRecord = await this.store.findRecord('guid', this.node?.id, {
+            include: 'custom_metadata',
+            resolve: false,
+        });
+
+        const metadataRecord =
+            (await guidRecord.customMetadata) as CustomFileMetadataRecordModel;
```

Comment:
Nit: This should be Custom *Item* MetadataRecordModel, not Custom *File* MetadataRecordModel